src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -82,7 +84,54 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+func logPanic(proxyAddr string, msg string) {
+	cfg, err := config.Init(packageVersion)
+	if err != nil {
+		panic("Error initializing config")
+	}
+
+	// Create the client-side proxy.
+	c := &client.Client{
+		// TODO: Next port in case the regular one hasn't been closed properly
+		Addr:         proxyAddr,
+		ReadTimeout:  0, // don't timeout
+		WriteTimeout: 0,
+	}
+	ready := make(chan bool)
+	go c.ListenAndServe(func() { ready <- true })
+	<-ready
+
+	if err := logging.Init(); err != nil {
+		panic("Error initializing logging")
+	}
+
+	ready = logging.Configure(proxyAddr, cfg.CloudConfigCA, cfg.InstanceId, version, revisionDate)
+	<-ready
+
+	log.Error(msg)
```

Comment:
So I think this can all just be the following right?

```go
cfg, err := config.Init(packageVersion)
if err != nil {
	panic("Error initializing config")
}
logging.Configure("", cfg.CloudConfigCA, cfg.InstanceId, version, revisionDate)
log.Error(msg)
logging.Flush()
_ = logging.Close()
```